docs/topics/testing/tools.txt

```diff
@@ -1119,7 +1119,7 @@ particular URL. Decorate your test class or test method with
 Multi-database support
 ----------------------
 
-.. attribute:: TransactionTestCase.multi_db
```

Comment:
`multi_db` should remain documented, if only a stub. `TransactionTestCase.databases` should have `.. versionadded:: 2.2`.

Reply:
Just to make sure I follow you. Should I reuse the `multi_db` text below for `databases` but add a `.. deprecated 2.2` annotation for a `.. attribute:: TransactionTestCase.multi_db` stub that mentions it what it uses to do and that `databases = '__all__'` is no equivalent? Ditto for `allow_database_queries`?